### ISSUE TITLE
OSM PH not recognized as detailed map by Garmin XT 5.52

### ISSUE THREAD

**Issue by wes3284** [maning/osmphgps#56]
Hi Sir Maning,

I would like to thank you for you and your teams effort for this project. It is a great help for many Garmin users here in the Phil. I have been using your map for almost a year now and it keeps on improving. Great job!

Everything is working great on my Garmin XT version 5.52 for my HU + GPS unit. The only thing that's bothering me a bit is that whenever I go to Where > Recent Finds in my GPS this error always appears: "Detailed maps are required to use this feature. Maps are unavailable when the device is in USB mass storage mode. Go to www.GarminAsus.com for more information." I'm not sure if this is a bug or not. I have tried using RoadGuide Maps before and I didn't encounter this error.

**Comment by maning** [comment]
Thank you for reporting. Do the other `Where >` options work?

**Comment by wes3284** [comment]
Yes, everything else is working great except for that tiny problem.

**Comment by wchen80** [comment]
hi wes3284, I have the same tiny problem like yours.  did you find a solution for that?
thks
